spec/acceptance/users_request_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 require "rspec_api_documentation/dsl"
 
-RSpec.resource "Api::V1::Users" do
+RSpec.resource "/users" do
```

Comment:
Can I provide an example of acceptance spec here?

I think this variant a little bit more clear because you don't need to override unnecessary attributes in all contexts. We override only attributes which are needed to check that response will incorrect. For example, when we check that request will be a failure "when no password provided", we need override only password, we do not need to change email. What do you think about it? If you agree can we rewrite all specs to make them look the same?

```ruby
require "rails_helper"
require "rspec_api_documentation/dsl"

RSpec.resource "Users" do
  include_context "with API request"

  post "/api/v1/users" do
    with_options scope: :user, with_example: true do
      parameter :email, required: true
      parameter :full_name
      parameter :password, required: true
    end

    let(:user_attributes) { { email: email, full_name: full_name, password: password } }
    let(:email) { "john.smith@example.com" }
    let(:full_name) { "John Smith" }
    let(:password) { "password" }

    let(:created_user) { User.last }

    let(:expected_response) do
      {
        "user" => {
          "id" => created_user.id,
          "fullName" => "John Smith",
          "email" => "john.smith@example.com"
        }
      }
    end

    example_request "Create user" do
      expect(response_status).to eq(201)
      expect(json_response).to eq(expected_response)
    end

    context "when email already taken" do
      before { create :user, email: "john.smith@example.com" }

      it_behaves_like "error response", "email" => ["taken"]
    end

    context "when email is invalid" do
      let(:email) { "" }

      it_behaves_like "error response", "email" => %w[blank invalid]
    end

    context "when no password provided" do
      let(:password) { "" }

      it_behaves_like "error response", "password" => ["blank"]
    end
  end
end
```